sotopia/database/utils.py

```diff
@@ -0,0 +1,178 @@
+import json
+
+import pandas as pd
+from pydantic import ConstrainedList, conlist, root_validator
+from redis_om import HashModel, JsonModel
+from redis_om.model.model import Field
+
+from .logs import EpisodeLog
+from .persistent_profile import AgentProfile, EnvironmentProfile
+
+
+class TwoAgentEpisodeWithScenarioBackgroundGoals(JsonModel):
```

Comment:
Does it need to be `redis_om.JsonModel`? Or just `pydantic.BaseModel`. I would also suggest removing `index=True` and make these non-optional.

Reply:
Oh why? I think all of our classes use jsonModel?

Reply:
JsonModel inherits pydantic.BaseModel with extra functions of interaction with redis. For example, "index=True" doesn't have any meaning here.